doc/client_lua_api.md

```diff
@@ -117,10 +117,13 @@ The main Lua script. Running this script should register everything it
 wants to register. Subsequent execution depends on Luanti calling the
 registered callbacks.
 
-**NOTE**: Client mods currently can't provide textures, sounds, or models by
+**NOTE**: Client mods currently can't provide textures, or models by
```

Comment:
This statement was true as-is: Client mods themselves can't provide sounds. You can't put sound files (or a `sounds` directory) in a client mod and have it work (as it would in a normal mod).

Reply:
Sure, but as there is currently no way to officially distribute and automatically install client-side mods, their installation is manual anyway and everything is thus possible :-)